sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/Infrastructure/TestData.cs

```diff
@@ -327,7 +327,7 @@ internal static class TestData
             },
             TimeGranularity.PerMinute);
 
-        public static UnivariateChangePointDetectionResult ExpectedChangePointResult = new UnivariateChangePointDetectionResult(
+        public static UnivariateChangePointDetectionResult ExpectedChangePointResult = AIAnomalyDetectorModelFactory.UnivariateChangePointDetectionResult(
```

Comment:
this actually does not change anything, the model factory also calls the internal ctor as we previously did